Project 3 Lol

```diff
@@ -0,0 +1,350 @@
+ - I think their point tracks on to relations like "coincident with" and "colocalizes with".
+ - A process or entity is always perfectly coincident or colocalized with itself because of its identity.
+  
+  (b)  Transitive 
+ - "immersed in" should be a transitive object property.
+ - If object a is immersed in fluid b, and fluid b is immersed in fluid c, then object a is immersed in fluid c.
+ - I am also interpreting "immersed in" as one type of the "located in" relation, which is listed as a transitive object property.
+ 
+  (c)  Symmetric
+ - I am confused about the "before or simultaneous with" relation because I do not understand why those relationships are not kept separate.
+ - "simultaneous with" should definitely be a symmetric temporal relation.
+  
+  (d)  Functional 
+ - I think "has prototype" should be a functional object property.
+ - If a prototype is "some instance of a 'canonical'" entity, I am assuming there is only one prototype for every entity of interest.
+ 
+  (e)  Symmetric and Reflexive
+ - "overlaps" seems symmetric and reflexive to me (I got help from the project 3 Google doc on this one).
+  
+```
+
+2. In BFO and RO identify at least one object property for each of a-e that _should not have the listed property, but which does_; argue for your case, using examples. 
+```
+  (a)  Irreflexive
+ - I'm a little confused about the irreflexive property of the "has member" relation.
+ - The relation is specifically designed to exist between items and collections, so what about a collection with only one item in it?
+ - Can an object not be classed both as an 'item' and as a 'collection' of only itself?
+ 
+  (b)  Transitive 
+ - I actually cannot find any relations listed in RO that I take issue with having the transitive object property.
+ - I do support Karl's justification for unlisting "ends after" as transitive due to the existence of time-traveling-to-kill-your-grandfather paradox.
+ - I do support Jaron's justification for unlisting "aligned with" as transitive due to the existence of non-transitive alignment relations like overlap and adjacency.
+ 
+  (c)  Asymmetric
+ -  "has role in modeling" is an asymmetric relationship between information about an entity and the entity of interest.
+ - I feel like if the entity of interest is also information, the two could hold a symmetric relationship.
+ - Like, does the information in the left circle of a venn diagram not have a role in modeling the right side?
+
+  (d)  Functional 
+ - "characteristic of" is a functional relationship between a characteristic and the relevant bearer.
+ - Isn't a functional object property limited to only a singular two-place relation? like, according to RO, a bearer can only have at most one characteristic?
+ - I am confused about this object property because I believe a bearer could have an infinite number of characteristics
+ 
```

Comment:
You are right, but "characteristic of" goes in the other direction, i.e. from the characteristic to the object. And in the framework we are using, we probably want to say that the colour x is characteristic of only one thing. (But I must admit that the notion of characteristic is still a bit opaque to me).